app/controllers/staff/reports_controller.rb

```diff
@@ -150,7 +150,7 @@ def approved_reports
 
   def send_csv
     response.headers["Content-Type"] = "text/csv"
-    response.headers["Content-Disposition"] = "attachment; filename=#{ERB::Util.url_encode(@report.description)}.csv"
+    response.headers["Content-Disposition"] = "attachment; filename=#{ERB::Util.url_encode(@report_presenter.financial_quarter_and_year)}-#{ERB::Util.url_encode(@report.description)}.csv"
```

Comment:
Do you think it is worth adding the fund title as well, that really makes the name complete for a DP without the description, we already have the `fund` in the ReportPresenter?